src/sortStudents.ts

```diff
@@ -1,16 +1,77 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function averageGrade(arr: number[]): number {
+  let sum: number = 0;
+
+  sum = arr.reduce((prev, item) => prev + item, 0);
+
+  return sum / arr.length;
+}
+
+export function sortStudents(
+  students: Student[], sortBy: SortType, order: SortOrder,
+): Student[] {
+  const copyOfStudents = [...students];
+  const typeOfOrder = order === 'asc';
+
+  switch (sortBy) {
+    case SortType.Name:
+      return typeOfOrder ? copyOfStudents.sort(
+        (a: Student, b: Student) => a.name.localeCompare(b.name),
```

Comment:
remove extra type & try to give the more clear name
use it in all cases
```suggestion
        (firstStudent, sesondStudent) => a.name.localeCompare(b.name),
```